src/purge.cls.php

```diff
@@ -797,6 +810,7 @@ public function purge_url($url, $purge2 = false, $quite = false)
 		self::add($hash, $purge2);
 
 		!$quite && !defined('LITESPEED_PURGE_SILENT') && Admin_Display::success(sprintf(__('Purge url %s', 'litespeed-cache'), $val));
+		do_action('litespeed_purged_link');
```

Comment:
@timotei-litespeed Maybe add the url as param?

Reply:
Added and looked over other functions and made a few changes

Reply:
Good try. There are still some places you may miss. Also some comments added